docs/user_guide/mitigation.rst

```diff
@@ -80,11 +81,408 @@ Reductions
+
+and one step further in the equivalent form:
+
+.. math::
+
+   - \text{metric}_a + r \cdot \text{metric}_* \leq 0 \; \forall a \\
+   r \cdot \text{metric}_a - \text{metric}_* \leq 0 \; \forall a
+
+To use a constraint in its ratio-based version simply indicate as much through
+its arguments
+
+    >>> ClassificationMoment(ratio_bound=0.9, ratio_bound_slack=0.01)
+
+In other words, this means that
+:math:`0.9 \cdot \text{metric}_a \leq \text{metric}_*`, i.e., the metric value for
+each group needs to be at least 90% of the overall metric value, and
+:math:`0.9 \cdot \text{metric}_* \leq \text{metric}_a`, i.e., the overall metric
+value needs to be at least 90% of each group's metric value.
+
+.. note::
+
+    It is not possible to specify both :code:`difference_bound` *and*
+    :code:`ratio_bound` for the same constraint object.
+
+.. _demographic_parity:
+
+Demographic Parity
+^^^^^^^^^^^^^^^^^^
+
+A classifier :math:`h(X)` satisfies *Demographic Parity* if
+
+.. math::
+    
+    \P[h(X) = y' \given A = a] = \P[h(X) = y'] \; \forall a, y'
+ 
+In other words, the selection rate or percentage of samples with label 1
+should be equal between all groups. Implicitly this means the percentage
+with label 0 is equal as well.
+
+In the example below class :code:`"a"` has a selection rate of 60%,
+:code:`"b"` has a selection rate of 20%. The overall selection rate is 40%,
+so :code:`"a"` is `0.2` above the overall selection rate, and :code:`"b"` is
+`0.2` below. The seemingly redundantly stored constraint violation with both
+:code:`"a"` and :code:`"b"` showing up twice stems from the constraint
+definition outlined in :ref:`constraints_binary_classification`. Each equality
+constraint is written as two inequalities of which one contains
+:math:`\text{metric}_a` positively (sign :math:`+`) and one negatively
+(sign :math:`-`).
+Note that the specified :code:`difference_bound` is not used
+since the constraint object is not used in a mitigation in this example.
+Similarly, :code:`y_true` is technically irrelevant to the calculations
+because the underlying metric of Demographic Parity, selection rate, does not
+consider performance relative to the true labels, but rather proportions in
+the predicted labels.
+
+.. note::
+
+    In mitigation examples we never need to call :code:`load_data` or
+    :code:`gamma` since the reductions techniques all automatically take care
+    of it. The example merely helps in illustrating the deviation from the
+    corresponding inequalities in :ref:`constraints_binary_classification`.
+
+.. doctest:: mitigation
+
+    >>> from fairlearn.reductions import DemographicParity
+    >>> from fairlearn.metrics import selection_rate_group_summary
+    >>> import numpy as np
+    >>> dp = DemographicParity(difference_bound=0.01)
+    >>> X                  = np.array([[0], [1], [2], [3], [4], [5], [6], [7], [8], [9]])
+    >>> y_true             = np.array([ 1 ,  1 ,  1 ,  1 ,  0,   0 ,  0 ,  0 ,  0 ,  0 ])
```

Comment:
Since y_true doesn't matter, I suggest you just put in y_pred, then specify (with a comment about not being needed) y_true = y_pred.

Reply:
I thought about that and decided not to do it, because the point here is to show what it does. Doing that would risk confusing readers for no reason other than avoiding the repetition of a line.

Reply:
Our API says `y_true` after all

Reply:
I had meant as a declaration of y_true, not in the actual method call itself. However, I do see your point.